lib/std/os/linux/syscalls.zig

```diff
@@ -0,0 +1,3505 @@
+// This file is automatically generated.
+// See tools/generate_linux_syscalls.zig for more info.
+pub const SYS = switch (@import("builtin").cpu.arch) {
+    .i386 => SysX86,
+    .x86_64 => SysX64,
+    .aarch64 => SysARM64,
+    .arm, .thumb => SysARM,
+    .riscv64 => SysRV64,
+    .sparc64 => SysSparc,
+    .mips, .mipsel => SysMips,
+    .powerpc => SysPPC,
+    .powerpc64, .powerpc64le => SysPPC64,
+    else => @compileError("unsupported architecture"),
```

Comment:
```suggestion
    else => @compileError("The Zig Standard Library is missing syscall definitions for the target CPU architecture"),
```

Reply:
I noticed that the `arch_bits` variable above has the same issue. Should a followup commit fix this or is it fine how it is?

Reply:
Since it's not part of your changeset, I'm happy with a follow-up commit. It's also low priority; I expect to making sweeping changes to address such concerns at some point as part of #1629.